pom.xml

```diff
@@ -17,105 +17,110 @@
     <url>https://github.com/liquibase/liquibase-redshift</url>
 
     <properties>
+        <maven.enforcer.requireJavaVersion>${maven.compiler.source}</maven.enforcer.requireJavaVersion>
         <liquibase.version>4.31.1</liquibase.version>
-        <jacoco.fileset.directory>target</jacoco.fileset.directory>
+        <liquibase.sdk.github.token>${env.GITHUB_TOKEN}</liquibase.sdk.github.token>
+        <sonar.tests>src/test/groovy</sonar.tests>
+        <junit.version>5.11.4</junit.version>
+        <junit-platform.version>1.11.4</junit-platform.version>
     </properties>
 
     <scm>
         <connection>scm:git:${project.scm.url}</connection>
         <developerConnection>scm:git:${project.scm.url}</developerConnection>
-	<url>https://github.com/liquibase/liquibase-redshift.git</url>
-	<tag>HEAD</tag>
+        <url>https://github.com/liquibase/liquibase-redshift.git</url>
+        <tag>HEAD</tag>
     </scm>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
-                <scope>test</scope>
-                <version>4.13.2</version>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
-
     <dependencies>
+        <dependency>
+            <groupId>net.bytebuddy</groupId>
+            <artifactId>byte-buddy</artifactId>
+            <version>1.17.2</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>com.amazon.redshift</groupId>
             <artifactId>redshift-jdbc42</artifactId>
             <version>2.1.0.32</version>
         </dependency>
+        <!-- AWS SDK dependencies needed by the Redshift JDBC driver -->
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-bundle</artifactId>
+            <version>1.12.656</version>
+        </dependency>
+        <!-- JSON libraries needed for test harness -->
+        <dependency>
+            <groupId>org.skyscreamer</groupId>
+            <artifactId>jsonassert</artifactId>
+            <version>1.5.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20240303</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
actually we shoudn't need junit-jupiter-api at all as junit-jupiter-engine has junit-jupiter-api as transitive dependency